### ISSUE TITLE
Reimport buttons possibly confuse new and seasoned users of Godot

### ISSUE THREAD

**Issue by Bat-Computer** [godotengine/godot-proposals#8793]
### Describe the project you are working on

This is a reference to either 2D or 3D projects. I'm raising this as it feels like a usability issue and can easily confuse users of Godot.

### Describe the problem or limitation you are having in your project

When a user makes changes to an imported 3D model or image, to save the changes the option available is 'Reimport'. This is confusing and counterintuitive to users as you would expect the button that saves changes would be titled 'Save'.

![Screenshot 2024-01-04 115831](https://github.com/godotengine/godot-proposals/assets/40317509/cf84cdf5-c188-4dd7-ba2b-262332639eed)

![Screenshot 2024-01-04 115934](https://github.com/godotengine/godot-proposals/assets/40317509/fd877ffb-e7dc-4a0b-ad15-21c1fc48427b)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

I haven't found information whether Godot works differently to other software with regards to making changes set and saved on files. In Uneal Engine if you make a change, you can Apply and save the changes, or you can click the item and select 'Save', it's much the same in Unity. If you save an alteration to an MS Word document, you would expect the file to be saved to the file, but not re-imported after the fact. 

Perhaps it's technically correct the way Godot does re-import/re-load a file after changes have been made for some technical reason that I'm not aware of, and I've been using godot for a while. 

But to the end user of Godot it's confusing that there is a button labelled 'Reimport' when you wish to save changes, no other software I've used has this button.

Perhaps it's labelled this way because the devs see that it is technically correct, but to a user like myself it is confusing and unnecessary.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Simply change the title of the button labelled 'Reimport' to 'Save' or 'Save Changes'. If it does indeed re-open/Re-import it is of little consequence to an end user.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

I'd imagine it is easy to change.

### Is there a reason why this should be core and not an add-on in the asset library?

It's just a label on a button ;)

**Comment by Bat-Computer** [comment]
I understand that some people wish to write scripts or do more advanced things, but for the average day-to-day user, who may just change an image to No Compression, or a user that just changes a 3D Model to have physics and a collision shape, pressing 'Reimport' seems counterintuitive and not that obvious. 

I have no problem with Godot being different, but usability wise, it's quite confusing I think. Change something, press 'Save' ;)

**Comment by AThousandShips** [comment]
But saying "save" is wrong? It isn't just saving, it does reimport, when *any* setting changes, that's the point. Have you found others who find this confusing? Some forum posts?

**Comment by Bat-Computer** [comment]
Yes, it's not just me, some online friends who've dabbled with Godot said the same and even my son who's questioned it numerous times.

I guess it's just not explained anywhere that I've seen, seems odd that it is necassary to re-import a file if a change is made. It's just not what you'd expect to happen in software generally. If you make a change to an asset, you would think that the current project retains the information while you're working, pressing save then allows the changes to be available when you restart the project.

Godot obviously works differently, but it just never stated why exactly in the docs, maube I missed it.

Of course if I'm being totally stupid here to more experienced users, sorry ;)

**Comment by AThousandShips** [comment]
That's anecdotal evidence tbh, I've never seen anyone confused about this, and in order to rename a button to something wrong we'd have to know a lot of users are confused

**Comment by AThousandShips** [comment]
I'd say this should be closed because this isn't a valid change, this is due to a confusion in what the button does, which isn't due to lacking documentation or general indications, the assumption that an import menu would be changing the original file isn't one based on the engine being confusing, or the documentation

**Comment by Bat-Computer** [comment]
I'm sorry as I stated, I didn't wish to upset anyone. It just didn't seem clear how it works. I've read the link in the docs, thanks for that, although for me it wasn't clear in it's explanation.

For me in Godot 3D projects I would typically import an FBX/GLTF add Physics and a Collision Shape in the 'Advanced Import Settings' window. At that stage I would then press the 'Reimport' button for the settings to be saved, or so I thought. I understood that the file would then be re-imported/Re-opened, as I said, I didn't understand why this was the case. 

I assumed (wrongly) that the changes I make on an asset in a game engine are saved to the oringal file on my PC, whilst those changes are also retiained while the project is open.

Some users don't always understand why something works the way it does, please don't be too critical.

With 2D Sprites I'd change the filter method

**Comment by AThousandShips** [comment]
Not upset, don't know how you got that impression

My confusion here is where you got these ideas, the term "importing" is well established in software and game development, and nowhere does it imply that pressing that button does what you assume it does, it says very clearly "reimport". The neutral and logical assumption from that is "it reimports the data"

So regardless of how confusing this process might be, the changes suggested here are completely wrong

So feel free to open a [documentation issue report](https://github.com/godotengine/godot-docs/issues) for this to have the process more clearly described, but changing the name would be wrong, and the button is very clear in what it does, it does exactly what it says 🙂

**Comment by Bat-Computer** [comment]
I think you misinterpreting what I said, I didn't say that the word Re-import was the same meaning as save, I understand the difference what the words mean ;)

I thought that it would be more clear to a user that their update/change was saved to the original file. The fact that Godot re-opens/Re-import the file isn't something it seemed that an end user would need to know. All they would need to know is that their change are saved - although it seems that this is more important that I considered.

I'm not the only one who stuggles with Godot's docs, many users do. It's not to say that they are bad, it's just that they assume a lot for new users, especially those that are not familiar with it's querks/differences. I teach and I see this all the time, sometimes how something is written, even with lots of detail, doesn't always convey the message that was intended in making it easy to understand. Anyway, I digress. Thanks for the information.

**Comment by AThousandShips** [comment]
> I thought that it would be more clear to a user that their update/change was saved to the original file ... All they would need to know is that their change are saved - although it seems that this is more important that I considered.

Yes, that's what I'm talking about, because that's not what happens, that's wrong, changes aren't saved, I was aware that you knew the words were different, the issue is that you want to change the button to say what it doesn't, instead of what it does do

And I agree, with the information, so that's why I said to open a documentation issue, but in this case the confusion isn't because of confusing interfaces, it's impossible to safeguard an interface from bad assumptions, the assumption is that the user takes the information at face value.

For example, should MS Word clarify when you save a word file that it won't update the pdf you exported before? Or should it safely assume you know it won't unless it says it will?
